Makefile

```diff
@@ -67,9 +68,11 @@ _save-refcache:
 
 check-links-all: check-links check-links-older check-links-external
 
-spellcheck:
-	cat scripts/cspell/project-names.txt | grep -v '^#' | grep -v '^\s*$$' | tr ' ' '\n' > scripts/cspell/project-names-parsed.txt
-	cd scripts/cspell && ./spellcheck.sh
+spellcheck-prep:
+	cat .cspell/project-names-src.txt | grep -v '^#' | grep -v '^\s*$$' | tr ' ' '\n' > .cspell/project-names.g.txt
```

Comment:
I think we can keep the name as project-names.txt, without src.

Reply:
You've kept the `-src` filename suffix in your suggested edits. So do you want me to drop `-src` or keep it?